cmd/tikv-ctl/src/main.rs

```diff
@@ -165,15 +163,15 @@ fn new_debug_client(host: &str, mgr: Arc<SecurityManager>) -> DebugClient {
 trait DebugExecutor {
     fn dump_value(&self, cf: &str, key: Vec<u8>) {
         let value = self.get_value_by_key(cf, key);
-        v1!("value: {}", escape(&value));
+        println!("value: {}", escape(&value));
```

Comment:
what's the difference between `v1!` and `println!`?

Reply:
`v1!` is from crate `vlog`. However as the PR descripted, here outputs are more like **result** instead of **log**, so I replace it with `println!`. And it's better to unify all logs to `slog` instead of keeping both `vlog` and `slog`.